CMake/RealmCore.cmake

```diff
@@ -109,6 +113,14 @@ function(download_realm_core core_version)
     set_property(TARGET realm PROPERTY IMPORTED_LOCATION_RELEASE ${core_library_release})
     set_property(TARGET realm PROPERTY IMPORTED_LOCATION ${core_library_release})
 
+    if (LINUX)
+        find_package(Threads)
```

Comment:
This should really be done unconditionally. The use of threads isn't specific to Linux.